lib/tasks/invite_code.rake

```diff
@@ -1,16 +1,21 @@
 
 
 desc "Generate a batch of invite codes"
-task :generate_invites, [:n] => :environment do |task, args|
+task :generate_invites, [:n, :u] => :environment do |task, args|
 
-    codes = args.n
-    codes = codes.to_i
+  codes = args.n
+  codes = codes.to_i
 
-    codes.times do |x|
+  if args.u != nil
+    max_uses = args.u
+  end
+
+  codes.times do |x|
     x = InviteCode.new
+    x.max_uses = max_uses
```

Comment:
Yes, if only n is given, the max uses will automatically be set to 1. 
I also included that in the documentation. :bowtie:

![bildschirmfoto 2015-10-13 um 17 03 50](https://cloud.githubusercontent.com/assets/10947846/10458732/8e75fb4e-71cc-11e5-88b5-c95aac5010b5.png)